app/experimenter/experiments/constants/nimbus.py

```diff
@@ -215,6 +215,86 @@ class NimbusTargetingConfig:
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_CASUAL_USER_URIS = NimbusTargetingConfig(
+    name="Casual user (uris)",
+    slug="casual_user_uris",
+    description="Between 7 and 13 URIs visited in the past 28 days",
+    targeting="userMonthlyActivity|length >= 7 && userMonthlyActivity|length <= 13 && "
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_REGULAR_USER_URIS = NimbusTargetingConfig(
+    name="Regular user (uris)",
+    slug="regular_user_uris",
+    description="Between 14 and 20 URIs visited in the past 28 days",
+    targeting="userMonthlyActivity|length >= 14 && userMonthlyActivity|length <= 20 && "
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_CORE_USER_URIS = NimbusTargetingConfig(
+    name="Regular user (uris)",
+    slug="regular_user_uris",
+    description="More than 20 URIs visited in the past 28 days",
+    targeting="userMonthlyActivity|length >= 20 && "
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 28",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_INFREQUENT_USER_TELEMETRY = NimbusTargetingConfig(
+    name="Infrequent user (telemetry)",
+    slug="infrequent_user_telemetry",
+    description="Infrequent users determined by telemetry",
+    targeting="mainPingSubmissions|length >= 1 && mainPingSubmissions|length <= 12 && "
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 60",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_CASUAL_USER_TELEMETRY = NimbusTargetingConfig(
+    name="Casual user (telemetry)",
+    slug="casual_user_telemetry",
+    description="Casual user determined by telemetry",
+    targeting="mainPingSubmissions|length >= 13 && mainPingSubmissions|length <= 26 &&"
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 60",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_REGULAR_USER_TELEMETRY = NimbusTargetingConfig(
+    name="Regular user (telemetry)",
+    slug="regular_user_telemetry",
+    description="Regular user determined by telemetry",
+    targeting="mainPingSubmissions|length >= 27 && mainPingSubmissions|length <= 40 &&"
+    "(currentDate|date - profileAgeCreated|date) / 86400000 >= 60",
+    desktop_telemetry="",
+    application_choice_names=(Application.DESKTOP.name,),
+)
+
+TARGETING_CORE_USER_TELEMETRY = NimbusTargetingConfig(
+    name="Regular user (telemetry)",
+    slug="regular_user_telemetry",
+    description="Regular user determined by telemetry",
```

Comment:
Should these be "Regular" or "Core" user? (Looks like a copy-pasta from L279-281 above)